src/test/java/com/kodilla/ecommercee/controller/UserControllerTest.java

```diff
@@ -0,0 +1,80 @@
+package com.kodilla.ecommercee.controller;
+
+
+import com.kodilla.ecommercee.domain.User;
+import com.kodilla.ecommercee.domain.UserDto;
+import com.kodilla.ecommercee.repository.UserRepository;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@Transactional
+public class UserControllerTest {
+
+    private static final String USER_NAME_KOWALSKI = "KOWALSKI JAKUB";
+
+    @Autowired
+    private UserController userController;
```

Comment:
Jeśli chcesz przetestować kontroler to należy użyć obiektu `MockMvc` i adnotacji `@WebMvcTest` nad klasą z testem